modules/logging-api-reference.adoc

```diff
@@ -0,0 +1,1155 @@
+// Module included in the following assemblies:
+//
+// * logging/
+
+:_content-type: REFERENCE
+[id="logging-api-reference_{context}"]
+= API Reference
+:toc:
+:toclevels: 4
+
+== ClusterLogForwarder
+
+[options="header"]
+|======================
+|Property|Type|Description
+|spec|object|  Specification of the desired behavior of ClusterLogForwarder
+|======================
+
+
+=== .spec
+===== Description
+ClusterLogForwarderSpec defines how logs should be forwarded to remote targets.
+
+=====  Type
+* object
+
+[options="header"]
+|======================
+|Property|Type|Description
+|inputs|array|  *(optional)* Inputs are named filters for log messages to be forwarded.
+|outputDefaults|object|  *(optional)* OutputDefaults are used to specify default values for OutputSpec
+|outputs|array|  *(optional)* Outputs are named destinations for log messages.
+|pipelines|array|  Pipelines forward the messages selected by a set of inputs to a set of outputs.
+|======================
+
+
+=== .spec.inputs[]
+===== Description
+InputSpec defines a selector of log messages.
+
+=====  Type
+* array
+
+[options="header"]
+|======================
+|Property|Type|Description
+|application|object|  *(optional)* Application, if present, enables named set of `application` logs that
```

Comment:
@jcantrill - 

> enables named set of `application` logs that .... 

Seems to have gotten truncated, what should this say?

Reply:
I don't know if there is a character limit or a break in the note that would keep us from getting the entire content